script/Common/00_00.txt

```diff
@@ -2032,17 +2037,17 @@ But that's okay.
 [sha:b3f91ef4d144ecf79e47dbe95c639a17b0884d06]{
 -- Page 154, Offset 495.
 -- 遠野志貴はひとりでもちゃんとやっていける。
-Tohno Shiki can do fine on his own.
+Tohno Shiki would do fine on his own.
```

Comment:
'would' makes it sound like this is future Shiki affirming retrospectively that he would indeed do fine on his own. If that's the case, the next line needs to be modified to fit that.